exec/internal/compile/backend_amd64.go

```diff
@@ -1171,10 +1202,26 @@ func (b *AMD64Backend) emitRHSConstOptimizedInstruction(builder *asm.Builder, re
 		prog.As = x86.AADDQ
 	case ops.I64Sub:
 		prog.As = x86.ASUBQ
+	case ops.I32Add:
```

Comment:
these new ops don't seem to be tested either.

Reply:
added test.